Common/Data/Custom/Benzinga/BenzingaMetadata.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using Newtonsoft.Json;
+
+namespace QuantConnect.Data.Custom.Benzinga
+{
+    /// <summary>
+    /// Article metadata
+    /// </summary>
+    public class BenzingaMetadata
+    {
+        /// <summary>
+        /// Is the article for pro subscribers only
+        /// </summary>
+        [JsonProperty("@pro")]
+        public bool IsPro { get; set; }
```

Comment:
It's all about how Json.NET deals with serializing XML to JSON. It adds a `#` prefix to the inner element. An example is `<a href="example">1234</a>` would show up as `#a` for `1234` and `@href`. 

I decided to go this way instead of using an auto-generated class from `xsd.exe` since I think it's a cleaner solution and bloats the codebase less (and makes it more understandable/maintainable too in my opinion)